frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -546,6 +546,7 @@ end
 function ReaderFooter:onPageUpdate(pageno)
     self.pageno = pageno
     self.pages = self.view.document:getPageCount()
+    self.ui.doc_settings:saveSetting("doc_pages", self.pages) -- for Book information
```

Comment:
I misunderstood the page count, if it changes time to time, my comment is not proper.